### ISSUE TITLE
Relative day displaying tomorrow instead of today

### ISSUE THREAD

**Issue by pith** [formatjs/formatjs-old#78]
I was expecting the following code to display "today" but it returns "tomorrow". I don't get why and how it works. Am I missing something or is it a real bug ?

```
<IntlProvider locale="en">
  <FormattedRelative 
    value={new Date('2018-06-15T17:00:00')} 
    units='day' 
    initialNow={new Date('2018-06-15T03:00:00')} 
  /> // => tomorrow
</IntlProvider>
```

**Comment by okuryu** [comment]
I guess this is a bug. It seems to occur if there is a difference of 11 hours 29 minutes 30 seconds on the same day.
```bash
$ node
> IntlRelativeFormat = require('intl-relativeformat')
{ [Function: RelativeFormat]
  defaultLocale: 'en',
  thresholds:
   { second: 45,
     'second-short': 45,
     minute: 45,
     'minute-short': 45,
     hour: 22,
     'hour-short': 22,
     day: 26,
     'day-short': 26,
     month: 11,
     'month-short': 11 },
  default: [Circular] }
> rf = new IntlRelativeFormat('en', {units: 'day'})
RelativeFormat { format: [Function: format] }
> rf.format(new Date('2012-12-12T11:29:29'), {now: new Date('2012-12-12T00:00:00')})
'today'
> rf.format(new Date('2012-12-12T11:29:30'), {now: new Date('2012-12-12T00:00:00')})
'tomorrow'
```

**Comment by longlho** [comment]
fixed in #45
